src/main/flight/mixer_init.c

```diff
@@ -370,7 +370,8 @@ void mixerInitProfile(void)
 void mixerResetRpmLimiter(void)
 {
     const float maxExpectedRpm = MAX(1.0f, motorConfig()->kv * getBatteryVoltage() * 0.01f);
-    mixerRuntime.rpmLimiterThrottleScale = constrainf(mixerRuntime.rpmLimiterRpmLimit / maxExpectedRpm, 0.0f, 1.0f);
+    const float rpm_derating = -5.44e-6 * maxExpectedRpm + 0.944;
```

Comment:
A comment about where this comes from might be a good idea here, since this is a empirical finding. Maybe something along the lines of `Empirical testing found this linear relationship between expected RPM and measured efficiency, independent from prop size`